crates/editor/src/editor_settings.rs

```diff
@@ -133,6 +135,20 @@ pub enum ShowScrollbar {
     Never,
 }
 
+/// When the show option permits it, which axes should show scrollbars.
+///
+/// Default: xy
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, JsonSchema, PartialEq, Eq)]
+#[serde(rename_all = "lowercase")]
+pub enum ScrollbarAxis {
```

Comment:
I'm not sold on naming axes as such; since it is user-surfaced it might be a bit confusing ("Is it YX or XY again?").
How about we spell it out verbosely and explicitly:
```
"axes": {
  "vertical": true,
  "horizontal: true
}
```

Reply:
Agreed, just pushed this